### PR TITLE
Move lesson and make it optional

### PR DESCRIPTION
**Pull request by kcleland0818**
This lesson will be optional and not required for students to implement in the project or lab.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,216 @@
...
+| getHour()                            | Gets the hour of the day                                                                   |
+| getMinute()                          | Gets the minute of the hour                                                                |
+| getMonth()                           | Gets the enum `Month` (e.g. Month.APRIL)                                                   |
+| getMonthValue()                      | Gets the integer representation of a month (e.g. 4 to represent April)                     |
+| getNano()                            | Gets the nanosecond field                                                                  |
+| getSecond()                          | Gets the second of the minute                                                              |
+| getYear()                            | Gets the year field                                                                        |
+| getZone()                            | Gets the timezone, such as 'EuropeParis'                                                   |
```
Comment (developerjaym) [inline README.md]: Should this be 'Europe/Paris'?

**Comment by developerjaym** [comment]
Great lesson!

Consider giving some use cases for ZonedDateTime. As an example, we used them at my last job to store when we received a work order. We were required to respond to emergency work orders within 4 hours, so it was important for us to know exactly when the order was received. Our biggest customer would only send us a 'store id' and a local datetime string, so we would have to look up the zone id for that store id and then create a ZonedDateTime from that local datetime string and the looked up zone id. Super annoying.

File: README.md (hunk 1 above)
Comment (kcleland0818) [reply README.md]: Yes! Thanks!

**Comment by kcleland0818** [comment]
> Great lesson!
> 
> Consider giving some use cases for ZonedDateTime. As an example, we used them at my last job to store when we received a work order. We were required to respond to emergency work orders within 4 hours, so it was important for us to know exactly when the order was received. Our biggest customer would only send us a 'store id' and a local datetime string, so we would have to look up the zone id for that store id and then create a ZonedDateTime from that local datetime string and the looked up zone id. Super annoying.

Playing with dates can definitely be annoying! Haha! Added some use cases for when to use ZoneDateTime or at least how they _could_ be applied in real-world applications. I tried to sum up your example within the one use case. I also gave my use case of how I had to work with ZonedDateTimes working on a military contract for DoD.

**Review by linda-seiter — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Pull request merged**